src/Authentication/Authentication.Test/Helpers/StringUtilTests.cs

```diff
@@ -1,37 +1,55 @@
-﻿using System;
-using System.Collections;
-using System.Collections.Generic;
-using System.Text;
+﻿namespace Microsoft.Graph.Authentication.Test.Helpers
+{
 
-using Microsoft.Graph.PowerShell.Authentication.Helpers;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Management.Automation;
 
-using Xunit;
+    using Microsoft.Graph.PowerShell.Authentication.Helpers;
+
+    using Xunit;
 
-namespace Microsoft.Graph.Authentication.Test.Helpers
-{
     public class StringUtilTests
     {
         private const string TestJsonArray = "{\"@odata.context\": \"https://graph.microsoft.com/v1.0/$metadata#users\"," + "\"value\": [{\"id\": \"6e7b768e-07e2-4810-8459-485f84f8f204\"},{\"id\": \"87d349ed-44d7-43e1-9a83-5f2406dee5bd\"}]}";
 
         private const string TestJsonObject = "{\"@odata.context\": \"https://graph.microsoft.com/v1.0/$metadata#users/$entity\",\"businessPhones\": [\"+1 412 555 0109\"],\"displayName\": \"Megan Bowen\",\"givenName\": \"Megan\",\"jobTitle\": \"Auditor\",\"mail\": \"MeganB@M365x214355.onmicrosoft.com\",    \"mobilePhone\": null,    \"officeLocation\": \"12/1110\",\"preferredLanguage\": \"en-US\",\"surname\": \"Bowen\",\"userPrincipalName\": \"MeganB@M365x214355.onmicrosoft.com\",\"id\": \"48d31887-5fad-4d73-a9f5-3c356e68a038\"}";
+        public static IEnumerable<object[]> TestJsonData =>
+            new List<object[]>
+            {
+                new object[] { TestJsonArray},
+                new object[] { TestJsonObject}
+            };
 
         [Theory]
         [MemberData(nameof(TestJsonData))]
         public void ShouldReturnCaseInsensitiveDictionaries(string jsonString)
         {
-            Exception ex = null;
-            var converted = jsonString.TryConvertToJson(out var jsonObj, ref ex);
+            var hashTable = jsonString.ConvertFromJson(true, null, out _);
 
-            Assert.True(converted);
-            Assert.IsType<Hashtable>(jsonObj);
-            var jsonHashTable = (Hashtable)jsonObj;
+            Assert.NotNull(hashTable);
+            var jsonHashTable = (Hashtable)hashTable;
             Assert.Equal(jsonHashTable["Value"], jsonHashTable["value"]);
         }
-        public static IEnumerable<object[]> TestJsonData =>
-            new List<object[]>
-            {
-                new object[] { TestJsonArray},
-                new object[] { TestJsonObject}
-            };
+
+        [Theory]
+        [MemberData(nameof(TestJsonData))]
+        public void ShouldReturnPsObject(string jsonString)
```

Comment:
this looks like it's testing if we call ConvertFromJson correctly we get the right object but not the actual system.  I think Peter enabled some integration tests so could we use those to hit the real code.

Reply:
Added Pester tests for all scenarios, see `src/Authentication/Authentication/test/Authentication.Tests.ps1`